src/histolab/filters/image_filters.py

```diff
@@ -43,6 +43,34 @@ def __call__(self, img: PIL.Image.Image) -> Union[PIL.Image.Image, np.ndarray]:
         return img
 
 
+class Lambda(object):
+    """Apply a user-defined lambda as a filter.
+
+    Inspired from:
+    https://pytorch.org/docs/stable/_modules/torchvision/transforms/transforms.html#Lambda
+
+    Parameters
+    ----------
+    lambd : callable
+        Lambda/function to be used as a filter.
+
+    Returns
+    -------
+    PIL.Image.Image
+        The image with the function applied.
+    """
+
+    def __init__(self, lambd: Callable[[PIL.Image.Image], PIL.Image.Image]) -> None:
+        assert callable(lambd), repr(type(lambd).__name__) + " object is not callable"
+        self.lambd = lambd
```

Comment:
maybe lmb istead of lambd?

Reply:
mmm I just copied it from torchvision, do you think lmb is better..?

Reply:
uh ok.... I just do not like lambda without the a.... but ok let's use torchvision style

Reply:
It's not possible to use lambda because it's a python keyword 🤷🏻‍♀️